docs/content.zh/release-notes/flink-1.16.md

```diff
@@ -0,0 +1,278 @@
+
+### Connectors
+
+#### Move Elasticsearch connector to external connector repository
+
+##### [FLINK-26884](https://issues.apache.org/jira/browse/FLINK-26884)
+
+The Elasticsearch connector has been copied from the Flink repository to its own individual repository
+at https://github.com/apache/flink-connector-elasticsearch
+
+#### Drop support for Hive versions 1.*, 2.1.* and 2.2.*
+
+##### [FLINK-27044](https://issues.apache.org/jira/browse/FLINK-27044)
+
+Support for Hive 1.*, 2.1.* and 2.2.* has been dropped from Flink. These Hive version are no longer
+supported by the Hive community and therefore are also no longer supported by Flink.
+
+#### Hive sink report statistics to Hive metastore
+
+##### [FLINK-28883](https://issues.apache.org/jira/browse/FLINK-28883)
+
+In batch mode, Hive sink now will report statistics for written tables and partitions to Hive metastore by default.
+This might be time-consuming when there are many written files. You can disable this feature by
+setting `table.exec.hive.sink.statistic-auto-gather.enable` to `false`.
+
+#### Remove a number of Pulsar cursor APIs
+
+##### [FLINK-27399](https://issues.apache.org/jira/browse/FLINK-27399)
+
+A number of breaking changes were made to the Pulsar Connector cursor APIs:
+
+- CursorPosition#seekPosition() has been removed.
+- StartCursor#seekPosition() has been removed.
+- StopCursor#shouldStop now returns a StopCondition instead of a boolean.
+
+#### Mark StreamingFileSink as deprecated
+
+##### [FLINK-27188](https://issues.apache.org/jira/browse/FLINK-27188)
+
+The StreamingFileSink has been deprecated in favor of the unified FileSink since Flink 1.12.
+This changed is reflected in the docs, but not yet in the codebase.
+
+#### Flink generated Avro schemas can't be parsed using Python
+
+##### [FLINK-2596](https://issues.apache.org/jira/browse/FLINK-25962)
+
+Avro schemas generated by Flink now use the "org.apache.flink.avro.generated" namespace for compatibility with the Avro Python SDK.
+
+#### Introduce Protobuf format
+
+##### [FLINK-18202](https://issues.apache.org/jira/browse/FLINK-18202)
+
+Apache Flink now support the Protocol Buffers (Protobuf) format. This allows you to use this format
+directly in your Table API or SQL applications.
+
+#### Introduce configurable RateLimitingStrategy for Async Sink
+
+##### [FLINK-28487](https://issues.apache.org/jira/browse/FLINK-28487)
+
+Supports configurable RateLimitingStrategy for the AsyncSinkWriter. This change allows sink implementers
+to change the behaviour of an AsyncSink when requests fail, for a specific sink. If no RateLimitingStrategy
+is specified, it will default to current default of AIMDRateLimitingStrategy.
+
+### Runtime & Coordination
+
+##### Remove brackets around variables
+
+###### [FLINK-24078](https://issues.apache.org/jira/browse/FLINK-24078)
+
+The keys in the map returned my MetricGroup#getAllVariables are no longer surrounded by brackets,
+e.g., <job_id> is now stored as job_id.
+
```

Comment:
```suggestion
```
This was never merged; the associated ticket is still open. It's recommended to only create the release notes _after_ migrating all remaining open tickets to 1.17.